include/openssl/dsa.h

```diff
@@ -141,10 +141,12 @@ int DSAparams_print_fp(FILE *fp, const DSA *x);
 int DSA_print_fp(FILE *bp, const DSA *x, int off);
 # endif
 
-# define DSS_prime_checks 50
+# define DSS_prime_checks 64
 /*
- * Primality test according to FIPS PUB 186[-1], Appendix 2.1: 50 rounds of
- * Rabin-Miller
+ * Primality test according to FIPS PUB 186-4, Appendix C.3. Since we only
+ * have 1 value here we set the number of checks to 64 which is the 128 bit
```

Comment:
Maybe replace "1 value"  by
    "one value"       or
    "a single value"   or
    "a single non-random value"

Reply:
I'd prefer it spelt out too.